alembic/versions/c03a9776da91_insert_into_role.py

```diff
@@ -0,0 +1,48 @@
+"""insert into role
```

Comment:
You can add these extra functionality in your initial migration.
Just delete the db, And add all these insert functions into that initial migration